airtime_mvc/application/controllers/EmbedController.php

```diff
@@ -54,4 +63,43 @@ public function playerAction()
         $this->view->availableMobileStreams = json_encode($availableMobileStreams);
         $this->view->availableDesktopStreams = json_encode($availableDesktopStreams);
     }
+
+    public function currentDayProgramAction()
+    {
+        $this->view->layout()->disableLayout();
+
+        $CC_CONFIG = Config::getConfig();
+
+        $this->view->css = Application_Common_HTTPHelper::getStationUrl() . "widgets/css/airtime-widgets.css?".$CC_CONFIG['airtime_version'];
+        $this->view->jquery = Application_Common_HTTPHelper::getStationUrl() . "widgets/js/jquery-1.6.1.min.js?".$CC_CONFIG['airtime_version'];
+        $this->view->jquery_custom = Application_Common_HTTPHelper::getStationUrl() . "widgets/js/jquery-ui-1.8.10.custom.min.js?".$CC_CONFIG['airtime_version'];
+        $this->view->widget_js = Application_Common_HTTPHelper::getStationUrl() . "widgets/js/jquery.showinfo.js?".$CC_CONFIG['airtime_version'];
+    }
+
+    public function weeklyProgramAction()
+    {
+        $this->view->layout()->disableLayout();
+
+        $CC_CONFIG = Config::getConfig();
+
+        $request = $this->getRequest();
+
+        $widgetStyle = $request->getParam('style');
+        if ($widgetStyle == "premium") {
+            $this->view->widgetStyle = "premium";
+            $this->view->css = Application_Common_HTTPHelper::getStationUrl() . "/css/embed/weekly-schedule-widget.css?" . $CC_CONFIG['airtime_version'];
+        } else {
+            $this->view->widgetStyle = "basic";
+            $this->view->css = Application_Common_HTTPHelper::getStationUrl() . "/css/embed/weekly-schedule-widget-basic.css?" . $CC_CONFIG['airtime_version'];
+        }
+        $this->view->jquery = Application_Common_HTTPHelper::getStationUrl() . "widgets/js/jquery-1.6.1.min.js?".$CC_CONFIG['airtime_version'];
```

Comment:
This is another thing we should fix at some point. Hardcoding jquery seems bad. Would be good if we had a factory for all these includes...